test/worlds/models/nested_models/model.sdf

```diff
@@ -0,0 +1,71 @@
+<?xml version='1.0'?>
+<sdf version='1.8'>
+
+  <model name="nested_models">
+    <model name="model_00">
+      <pose>0 0 0 0 0 0</pose>
+      <frame name="frame_00" attached_to="model_01::frame_01">
+        <pose>10 11 12 0 0 0</pose>
+      </frame>
+      <link name="link_00">
+        <pose relative_to="model_01::frame_01">0 0 0 0 0 0</pose>
+        <collision name="collision_00">
+          <geometry>
+            <sphere>
+              <radius>0.5</radius>
+            </sphere>
+          </geometry>
+        </collision>
+        <visual name="visual_00">
+          <geometry>
+            <sphere>
+              <radius>0.5</radius>
+            </sphere>
+          </geometry>
+        </visual>
+      </link>
+
+      <model name="model_01">
+        <frame name="frame_01">
+          <pose> 20 21 22 0 0 0</pose>
+        </frame>
+        <link name="link_01">
+          <pose relative_to="frame_01">0 0 0 0 0 0</pose>
+          <collision name="collision_01">
+            <geometry>
+              <box>
+                <size>1 1 1</size>
+              </box>
+            </geometry>
+          </collision>
+          <visual name="visual_01">
+            <geometry>
+              <box>
+                <size>1 1 1</size>
+              </box>
+            </geometry>
+          </visual>
+        </link>
+
+        <joint name="joint_01" type="continuous">
+          <parent>frame_00</parent>
```

Comment:
I was getting the error `Failed to resolve parent link for joint 'joint_01' with parent name 'frame_00'` when I ran  `INTEGRATION_sdf_frame_semantics`. This is because `frame_00` can't be referenced from this scope.